frontend/src/components/TileContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import { MouseEvent, ReactElement, useState } from 'react';
+
+import { TILE_BOARD_BRIGHT_SOURCE, TILE_BOARD_DARK_SOURCE } from '../constants/layoutElements';
+import TileState from '../constants/tileState';
+import Tile from '../model/Tile';
+
+const idleImagePath = TILE_BOARD_DARK_SOURCE;
+const activeImagePath = TILE_BOARD_BRIGHT_SOURCE;
+interface TileInterface {
+  tile: Tile;
+}
+
+const TileContainer = (props: TileInterface): ReactElement => {
+  const { tile } = props;
+  const TileImageSource = tile.getTileImage();
```

Comment:
variable should be camelCase... but we do not need it here - invoke tile.getTileImage()  from src